### DISCUSSION TITLE
Low Detail - Hide Higher Planes

### DISCUSSION THREAD

**Discussion by 6enny6** [runelite/runelite#16204] [Q&A] [answered]
Is it possible to add a ‘hide higher planes’ option to the Low Detail plugin? 

I’m hoping this would remove the tall pillars on the corners of rooms inside CG

**Comment by Enriath** [comment, marked as answer]
In the vanilla settings, you can disable all roofs at all times. Jagex did this by removing all the higher planes, so that would do what you want. If the pillars are still there, they have nothing to do with higher planes then.
Also to note, if you want to try this, disable the `Roof Removal` plugin, as that overrides the vanilla behaviour to make the roof hiding better.
